autovalet_husky/launch/aruco_harware.launch

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
filename typo. `harware` instead of `hardware`

Reply:
removed the file.